src/main/java/core/AbstractGameState.java

```diff
@@ -106,41 +93,72 @@ void reset(long seed) {
         reset();
     }
 
+    public final void setPlayerResult(Utils.GameResult result, int playerIdx) {
```

Comment:
Documentation issue: with methods shuffled around in the class (which is also really confusing Git and making it hard to tell what's actually changed/removed/added), the comments are no longer applying (and I'd like to keep them valid, so having getters / setters grouped, for example)

Reply:
Not a problem - I'll reorganise.